data/transport missions.txt

```diff
@@ -1868,6 +1872,309 @@ mission "Terraforming 5"
+				accept
+	
+	on enter "Cebalrai"
+		dialog
+			`Your ship is targeted by pirate fleets when you enter the system. Before you engage, you receive a message from Amy.`
+			`"Captain, these pirate ships have been occupying the system for days. You must take them out if we are to safely crash an asteroid into Tundra."`
+	npc evade
+		government "Bounty Hunter"
+		personality nemesis staying target
+		system "Cebalrai"
+		fleet "Marauder fleet I"
+		fleet "Large Southern Pirates"
+
+
+
+mission "Terraforming 12"
+	landing
+	name "Terraforming Tundra"
+	description "Plant a thruster on an asteroid to guide it into a collision with the dormant supervolcano on Tundra."
+	cargo "thruster equipment" 10
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Terraforming 11: done"
+	
+	on offer
+		payment 300000
+		conversation
+			`The pirates that occupied the system did not make an effort to land on Tundra, but they did harass any merchant ships that passed through the system. The government of Tundra pays you <payment> for ridding the system of the pirates, a pleasantly unexpected reward.`
+			`	You gather with the scientists in the spaceport bar. The Tundra native informs you that the area around the impact zone has been evacuated. "All we need to do now is find the right asteroid" After only a short time of searching, Amy finds the perfect candidate.`
+			`	"Just like last time, Captain," she says. Spaceport workers load a small thruster and the equipment you will need to mount it on the asteroid into your ship.`
+				accept
+	
+	npc assist
+		government "Uninhabited"
+		personality derelict fleeing uninterested waiting pacifist mute
+		ship "Asteroid" "Target Asteroid"
+	
+	on complete
+		payment 200000
+		event "terraforming timer 2" 365 730
+		event "terraforming Tundra"
+		conversation
+			`A plume of smoke forms in a mushroom cloud shape above where the asteroid hit. The scientists are viewing their instruments in anticipation for what happens next.`
+			`	Almost an hour after the impact, the ground begins to shake slightly, even though the volcano is located hundreds of kilometers away. "This might be it!" Nolan exclaims as he watches closely at a seismometer receiving readings from near the impact site. Suddenly the seismometer spikes, and another plume of smoke, mucher larger than the asteroid impact, rises from the impact site. Half an hour later, the low sound of the massive eruption can be heard in the spaceport, triggering another burst of cheers from the scientists. "It actually worked!" one of them yells out.`
+			`	You fly the scientists over top of the volcano, allowing them to take readings of the air and surface. "Atmospheric carbon dioxide levels are at 4000 parts per million. Methane levels at 20 parts per million. As the smoke disperses across the planet, those levels should drop to a tenth of what they are now, warming the entire planet."`
```

Comment:
Maybe instead of saying what the highest levels are, and mentioning the drop, you should phrase the change in terms of the rise. Skimmers will see "eruption -> high numbers -> dropping numbers -> warmer planet" and not immediately reconcile that this drop actually means the numbers increased everywhere else., and *that* increase is what is causing the warming effect.